scripts/poeditor/app-store/zh.json

```diff
@@ -0,0 +1,195 @@
+{
+    "APP_STORE_NEW_FEATURES": "小改进与错误修补",
+    "APP_STORE_FULL_DESCRIPTION": "了解并减少碳足迹。 记录您的碳排放指标并开始策划行动以达到每个月的最低标准。",
+    "APP_STORE_SHORT_DESCRIPTION": "碳排放记录与持续性指南",
+    "APP_STORE_KEYWORDS": "气候, 气候变化, 补偿, 抵消, 计算, 二氧化碳, 碳, 零废弃, 素食, 脚印",
```

Comment:
the first 5 lines / 4 items are relevant here, the rest can be deleted since this file is about the app store